lib/containers/common.pm

```diff
@@ -69,21 +66,18 @@ sub install_docker_when_needed {
             if ($host_os eq 'centos') {
                 assert_script_run "dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo";
                 # if podman installed use flag "--allowerasing" to solve conflicts
-                assert_script_run "dnf -y install docker-ce --nobest --allowerasing", timeout => 120;
+                assert_script_run "dnf -y install docker-ce --nobest --allowerasing", timeout => 300;
```

Comment:
Out of curiosity, why are you installing docker from docker's official repo and not from the CentOS repository? I would expect that the usual user would just `dnf install moby-engine` instead of adding an extra repository.

Reply:
It comes from the official documentation...
https://docs.docker.com/engine/install/centos/

And it's very problematic, as docker and podman can't be together in Centos since they have some dependency issues in containerd packages... very ugly... I would personally disable testing CentOS+docker... but we can't.

Reply:
i think @dcermak is right. the docker is `moby-engine` now.

Reply:
I have tried, CentOS doesn't come with moby-engine in default repos. 
I leave it up for investigation and see if it's worth it. Will create ticket for that, since it might be a good move.

Reply:
@b10n1k It is the case for Fedora.
@jlausuch sorry, I looked over pkgs.org too quickly. `moby-engine` is available for centos, but only from Microsoft's repository. So forget about my suggestion.